modules/cli-installing-cli.adoc

```diff
@@ -210,7 +210,7 @@ ifndef::openshift-origin,microshift[]
 +
 [NOTE]
 ====
-For macOS arm64, choose the *OpenShift v{product-version} macOS arm64 Client* entry.
+For macOS arm64, choose the *OpenShift v{product-version} macOS aarch64 Client* entry.
```

Comment:
@juliemathew @pravin-dsilva request you to please provide your inputs on above change. Thanks.

Reply:
If this was for linux, I think this change would be correct. However, apple and macOS only uses the `arm64` format. So I believe it should just be `macOS arm64 Client`

Reply:
@kelbrown20 thank you for the clarification. I have updated the PR as per above. Thanks.

cc: @manojnkumar @juliemathew @pravin-dsilva